lms/djangoapps/grades/tests/test_scores.py

```diff
@@ -47,20 +51,20 @@ class TestGetScore(TestCase):
     display_name = 'test_name'
     location = 'test_location'
 
-    SubmissionValue = namedtuple('SubmissionValue', 'exists, weighted_earned, weighted_possible')
-    CSMValue = namedtuple('CSMValue', 'exists, raw_earned, raw_possible')
+    SubmissionValue = namedtuple('SubmissionValue', 'exists, points_earned, points_possible, created_at')
```

Comment:
I'm not too thrilled on renaming these variables here.  Not distinguishing between `raw` and `weighted` has bit us several times and this is why this code explicitly chose to name the variables as such in this file.  

Yes, it differs from what the Submissions API returns, but that's why the mapping existed in `_create_submissions_scores`.  I'd rather keep the naming as it was, and just localize the names of `points_earned` and `points_possible` to the `_create_submissions_scores` method.   This will keep the explicit reminder that the Submissions API returns the `weighted` values and not the `raw` ones.